ClosedXML_Tests/Excel/Saving/SavingTests.cs

```diff
@@ -575,5 +575,19 @@ public void RemoveExistingInlineStringsIfRequired()
                 }, @"Other\InlineStrings\outputfile.xlsx");
             }
         }
+
+        [Test]
+        public void CanSaveFileWithEmptyFill()
+        {
+            // Arrange
+            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"TryToLoad\EmptyFill.xlsx")))
+            using (var wb = new XLWorkbook(stream))
+            using (var ms = new MemoryStream())
+            {
+                // Act
+                wb.SaveAs(ms, false);
+            }
+                // Assert
```

Comment:
Though I understand that you are checking that `SaveAs` does not throw, test without Assert looks unfinished. Better make you intention explicit:
```
Assert.DoesNotThrow(() => wb.SaveAs(sa, false));
```